hail/python/hailtop/hailctl/dataproc/cli.py

```diff
@@ -191,6 +191,12 @@ def start(
     debug_mode: Ann[
         bool, Opt(help='Enable debug features on created cluster (heap dump on out-of-memory error)')
     ] = False,
+    public_ip_address: Ann[
+        bool,
+        Opt(
+            help='Allow nodes to have a public IP address, and hence make requests on the public internet (default is internal-only from dataproc 2.2).'
```

Comment:
Good point, looks like we get the list of dependencies from [`hail/python/hailtop/hailctl/deploy.yaml`](https://github.com/hail-is/hail/blob/main/hail/python/hailtop/hailctl/dataproc/start.py#L276), which is [generated](https://github.com/hail-is/hail/blob/main/hail/Makefile#L306) when we build the wheel, so there is not a way for the user to avoid using the packages we specify. Revised to just make this part of the command string instead of an option!